inst/rmarkdown/templates/transmissibility/skeleton/skeleton.Rmd

```diff
@@ -441,25 +467,11 @@ ggplot(
 # Growth rate ($r$) and reproduction number ($R$)
 
 ```{r}
-last_date <- dat %>%
-  pull(date) %>%
-  max()
-
-# version using keep_first and keep_last from i2extras
-days_to_keep <- params$incomplete_days + params$r_estim_window
-i_recent <- dat_raw %>%
-  incidence("date",
-    counts = count_var,
-    groups = group_var
-  ) %>%
-  keep_last(days_to_keep) %>% # keep data for fitting
-  keep_first(params$r_estim_window) # remove incomplete data
-```
-
-```{r}
+# This code creates a dataset with daily incidence data, which is needed for
+# Rt estimation
 dat_i_day <- dat_raw %>%
   incidence("date",
-    interval = "daily",
+    interval = 1L,
```

Comment:
The reason is that when using "daily" instead of 1L I was getting an error message with incidence2 and I wasn't able to run the script, I thought maybe it was because this package was updated?